core-bundle/src/Resources/contao/pages/PageRedirect.php

```diff
@@ -31,7 +32,9 @@ public function generate($objPage)
 
 		$this->prepare($objPage);
 
-		$this->redirect(System::getContainer()->get('contao.insert_tag.parser')->replaceInline($objPage->url), $this->getRedirectStatusCode($objPage));
+		$url = UrlUtil::makeAbsolute(System::getContainer()->get('contao.insert_tag.parser')->replaceInline($objPage->url), Environment::get('path'));
```

Comment:
I think this needs to be one of:
```suggestion
		$url = UrlUtil::makeAbsolute(System::getContainer()->get('contao.insert_tag.parser')->replaceInline($objPage->url), Environment::get('base'));
```

```suggestion
		$url = UrlUtil::makeAbsolute(System::getContainer()->get('contao.insert_tag.parser')->replaceInline($objPage->url), Environment::get('path') . '/');
```

Reply:
You are right. While `UrlUtil::makeAbsolute` creates an absolute path anyway if `Environment::get('path')` is an empty string here:

https://github.com/contao/contao/blob/2082fff2c24229caf89dbfcd5afd89ffcb4fa33d/core-bundle/src/Util/UrlUtil.php#L38

the absolute URL will be wrong if `Environment::get('path')` is something like `/c413/public`.